packages/client/components/contacts/ContactsForm.tsx

```diff
@@ -181,6 +196,19 @@ export default function ContactsForm(props: Props) {
                             isPrefilledAddContact={true}
                         />
                     )}
+                    {props.contact ? (
+                        <Button
+                            variant="outline"
+                            _text={{ color: "error.500" }}
+                            onPress={handleDeleteContact}
+                            testID="deleteContactButton"
+                            style={styles.deleteContactButton}
+                        >
+                            Delete contact
+                        </Button>
+                    ) : (
+                        <></>
+                    )}
```

Comment:
You can also switch it from a ternary because there is no negative case
```suggestion
                    {props.contact && (
                        <Button
                            variant="outline"
                            _text={{ color: "error.500" }}
                            onPress={handleDeleteContact}
                            testID="deleteContactButton"
                            style={styles.deleteContactButton}
                        >
                            Delete contact
                        </Button>
                    )}
```

Reply:
Add `marginTop={"35px"}` to fix the spacing and remove `styles.deleteContactButton`.